AI-and-Analytics/Getting-Started-Samples/INC-Quantization-Sample-for-PyTorch/README.MD

```diff
@@ -64,6 +54,7 @@ The sample contains one Jupyter Notebook and one Python script.
 
 ## Run the `Getting Started with Intel® Neural Compressor for Quantization` Sample
```

Comment:
```suggestion
## Run the `Getting Started with Intel® Neural Compressor for Quantization` Sample
```If you have already set up the PIP or Conda environment and installed AI Tools go directly to Run the Notebook.